graphql_compiler/schema_generation/sqlalchemy/exceptions.py

```diff
@@ -0,0 +1,6 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from ..exceptions import SchemaError
+
+
+class InvalidSQLEdgeReferenceError(SchemaError):
+    """Raised when a SQL edge references a non-existent vertex or column."""
```

Comment:
1. Since you decided to make a separate file for sqlalchemy exceptions, why not do the same for orientdb? `..exceptions` contains some orientdb-specific ones.

Reply:
I'll remove this file and change the exception to InvalidClassError. It makes sense.